packages/mds-api-server/api-server.ts

```diff
@@ -42,25 +42,26 @@ export const ApiServer = (
   // Middleware
   app.use(
     CompressionMiddleware(options.compression),
-    RequestLoggingMiddleware(options.requestLogging),
     CorsMiddleware(options.cors),
     JsonBodyParserMiddleware(options.jsonBodyParser),
     MaintenanceModeMiddleware(options.maintenanceMode),
-    AuthorizationMiddleware(options.authorization)
+    AuthorizationMiddleware(options.authorization),
+    /** Prometheus Middleware
+     * Placed after the other middleware so it can label metrics with additional
+     * properties added by the other middleware.
+     */
+    PrometheusMiddleware(options.prometheus),
+    /** Request Logging Middleware
+     * Placed after Prometheus middleware to avoid excessive logging
+     */
+    RequestLoggingMiddleware(options.requestLogging),
+    /** HTTP Context Middleware
+     * Placed after the other middleware to avoid causing collisions
+     * see express-http-context's README for more information
+     */
```

Comment:
I think moving this may result in requests coming in during maintenance mode being responded to prior to hitting the request logging middleware, so we won't get any logs for those

Reply:
Does that matter? We could move the `MaintenaceModeMiddleware` down meaning we could still hit `GET /prometheus` even when everything else is returning 503.

Reply:
Good call. I moved the `MaintenanceModeMiddleware` after `RequestLoggingMiddleware` so we'll still get logs for all the requests that are being 503'd.